src/BotwinExtensions.cs

```diff
@@ -130,6 +139,11 @@ private static void ApplyGlobalBeforeHook(IApplicationBuilder builder, BotwinOpt
             }
         }
 
+        /// <summary>
+        /// Adds Botwin to the specified <see cref="IServiceCollection"/>.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> to add Botwin to.</param>
+        /// <param name="assemblies">Optional assemblies <see cref="Assembly"/> to add to the services collection. If assemblies are not provided, Assembly.GetEntryAssembly is called.</param>
```

Comment:
`Optional array of <see cref="Assembly"/> to add to the services collection. If assemblies are not provided, Assembly.GetEntryAssembly is called.`